packages/evm/contracts/payments/PaymentsBook.sol

```diff
@@ -520,19 +475,19 @@ contract PaymentsBook is ControlledRecoverable, TimeHelpers, IPaymentsBook {
         }
 
         // Note that we already checked the guardian active balance is greater than zero.
-        // Then, the total active balance must be greater than zero.
+        // Then, the total active balance should be greater than zero.
         return _period.guardiansShares[_token].mul(_guardianActiveBalance) / _totalActiveBalance;
     }
 
     /**
-    * @dev Check if a given guardian has already claimed the owed share for a certain period
+    * @dev Tell if a guardian can claim the owed share for a certain period
     * @param _period Period being queried
     * @param _guardian Address of the guardian being queried
     * @param _token Address of the token to be queried
-    * @return True if the guardian has already claimed their share
+    * @return True if the guardian can claim their share
     */
-    function _hasGuardianClaimedShare(Period storage _period, address _guardian, address _token) internal view returns (bool) {
-        return _period.claimedGuardians[_guardian][_token];
+    function _canGuardianClaim(Period storage _period, address _guardian, address _token) internal view returns (bool) {
```

Comment:
Ah yes, were we going to re-flip this and the governor one back to `_hasGuardianClaimedShare()`?